linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/no_tmux_in_shells/ansible/shared.yml

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: {{{ rule_title }}} - Ensure tmux line not exists in /etc/shells
+  ansible.builtin.lineinfile: 
```

Comment:
Per the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#yaml) please use 4 spaces in new YAML files.

Reply:
i updated the solution, thanks for the review